var/spack/repos/builtin/packages/must/package.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Must(CMakePackage):
+    """MUST detects usage errors of the Message Passing Interface (MPI)
+    and reports them to the user. As MPI calls are complex and usage
+    errors common, this functionality is extremely helpful for application
+    developers that want to develop correct MPI applications. This includes
+    errors that already manifest: segmentation faults or incorrect results
+    as well as many errors that are not visible to the application developer
+    or do not manifest on a certain system or MPI implementation."""
+
+    homepage = "https://www.i12.rwth-aachen.de/go/id/nrbe"
+    url = "https://hpc.rwth-aachen.de/must/files/MUST-v1.8.0-rc1.tar.gz"
+
+    maintainers = ["jgalarowicz", "dmont"]
+
+    version(
+        "1.8.0-rc1",
+        sha512="ea30291ff5de2a43b225561ce65c61863a7afc5a127af0cc7b149d77574308981290a0913e1aa5547d88739bf7d56ae35cb29d7d164b175958ca4542f319da5b",
+    )
```

Comment:
Why specify `sha512` instead of `sha256` here?  Most of spack (including commands like `spack checksum`) assumes `sha256` ATM, so:

```suggestion
    version(
        "1.8.0-rc1",
        sha256="49fd2487fbd1aa41f4252c7e37efebd3f6ff48218c88e82f34b88d59348fe406",
    )
```